scss/components/_Modal.scss

```diff
@@ -23,14 +23,15 @@ $modal-content-padding: 0 !default;
   @include flex(center, row, center, nowrap);
   @include global-transition(opacity);
   opacity: 0;
-  z-index: $above-everything;
+  z-index: $below-everything;
   &.rev-Modal--closed {
     // so we can move the modal out of the way so that it doesn't intercept
     // clicks while closed
     right: 9999vw;
   }
   &.rev-Modal--open {
     opacity: 1;
+    z-index: $above-everything;
```

Comment:
@CheetoMao I made this change because in non-react environments, if the Modal was on the page, but not visible, it made it so nothing could be clicked. So I changed it to be below everything when not open and above when opened. Does this look like a good change to you?

Reply:
@bryanjos seems like a good change to me, but also kind of negates the need for lines 27-31, no?

Reply:
You are correct. It didn't look like it was being used either

Reply:
@CheetoMao I reverted this change and instead made sure that `rev-Modal--closed` is applied to a closed Modal